app/views/character_weapons/edit.html.erb

```diff
@@ -0,0 +1,44 @@
+<h2>Edit Weapon</h2>
```

Comment:
Watch out for your indentation, please review your tab settings are in set as 2 instead of 4 spaces